code/game/objects/items/storage/secure.dm

```diff
@@ -156,8 +156,7 @@
 
 /obj/item/storage/secure/briefcase/syndie/PopulateContents()
 	..()
-	var/datum/component/storage/storage_space = GetComponent(/datum/component/storage)
-	for(var/i in 1 to storage_space.max_items - 2)
+	for var/iterator in 5
```

Comment:
_What is this syntax_?

Reply:
HHHHHHHHHHHHHHHH

Reply:
🤨

Reply:
Too much matlab makes arcane a blind boy